lld/test/ELF/relro-data-unlikely.s

```diff
@@ -0,0 +1,50 @@
+# REQUIRES: x86
```

Comment:
keep-text-section-prefix.s

(Make the filename more generic in case this feature applies to `.data.unlikely` in the future)